AdblockPlusSafari/AdblockPlus.m

```diff
@@ -73,10 +81,40 @@ - (instancetype)init
     _enabled = [_adblockPlusDetails boolForKey:AdblockPlusEnabled];
     _acceptableAdsEnabled = [_adblockPlusDetails boolForKey:AdblockPlusAcceptableAdsEnabled];
     _activated = [_adblockPlusDetails boolForKey:AdblockPlusActivated];
+    _defaultFilterListEnabled = [_adblockPlusDetails boolForKey:AdblockPlusDefaultFilterListEnabled];
     _filterLists = [_adblockPlusDetails objectForKey:AdblockPlusFilterLists];
     _installedVersion = [_adblockPlusDetails integerForKey:AdblockPlusInstalledVersion];
     _downloadedVersion = [_adblockPlusDetails integerForKey:AdblockPlusDownloadedVersion];
     _whitelistedWebsites = [_adblockPlusDetails objectForKey:AdblockPlusWhitelistedWebsites];
+
+    // Migrate to new filter lists format
+
+    NSDictionary *defaultFilterListsFileNames =
+    @{DefaultFilterListName:
+        @"easylist_content_blocker.json",
+      DefaultFilterListPlusExceptionRulesName:
+        @"easylist+exceptionrules_content_blocker.json"
+      };
+
+    for (NSString *defaultFilterListName in defaultFilterListsFileNames) {
```

Comment:
I decided, that it would be more convenient to map filter lists by names rather then urls (which can be changed).